tests/Doctrine/Tests/ORM/Functional/EntityRepositoryTest.php

```diff
@@ -683,7 +684,7 @@ public function testCanRetrieveRepositoryFromClassNameWithLeadingBackslash()
     /**
      * @group DDC-1376
      *
-     * @expectedException Doctrine\ORM\ORMException
+     * @expectedException Doctrine\ORM\Repository\InvalidFindByInverseAssociation
      * @expectedExceptionMessage You cannot search for the association field 'Doctrine\Tests\Models\CMS\CmsUser#address', because it is the inverse side of an association.
```

Comment:
I just realized it now, that one test is using `$this->expectException()` and ` $this->expectExceptionMessage()` while  we have annotations here.

Reply:
I think annotations should be completely dropped in favor of those method calls :P